abi/constants.go

```diff
@@ -4,3 +4,4 @@ const trueAsByte = uint8(1)
 const falseAsByte = uint8(0)
 const optionMarkerForAbsentValue = uint8(0)
 const optionMarkerForPresentValue = uint8(1)
+const pubKeyLength = 32
```

Comment:
For the moment, we've promoted this to a constant. When the need arises, we might pass it as a parameter to an eventual `ValuesFactory`.